staging/src/k8s.io/client-go/rest/connection_test.go

```diff
@@ -54,14 +54,14 @@ func (lb *tcpLB) handleConnection(in net.Conn, stopCh chan struct{}) {
 	go io.Copy(in, out)
 	<-stopCh
 	if err := out.Close(); err != nil {
-		lb.t.Fatalf("failed to close connection: %v", err)
+		lb.t.Errorf("failed to close connection: %v", err)
```

Comment:
Was there a staticcheck issue with the use of `Fatalf`? I'm just wondering since using `Errorf` will lead to different behavior - the test will continue to run with `Errorf`, as opposed to exiting immediately with `Fatalf`.